easybuild/tools/docs.py

```diff
@@ -160,3 +147,134 @@ def avail_easyconfig_params(easyblock, output_format):
         FORMAT_TXT: avail_easyconfig_params_txt,
     }
     return avail_easyconfig_params_functions[output_format](title, grouped_params)
+
+def generic_easyblocks(path_to_examples, common_params={}, doc_functions=[]):
+    """
+    Compose overview of all generic easyblocks
+    """
+    modules = import_available_modules('easybuild.easyblocks.generic')
```

Comment:
the package name `easybuild.easyblocks.generic` should become a parameter of this function, so we can generate an overview of software-specific (i.e. non-generic) easyblocks too

maybe rename function to `gen_easyblocks_overview_rst`?